src/main/java/microsoft/exchange/webservices/data/core/EwsUtilities.java

```diff
@@ -1212,11 +1113,10 @@ public static void validateParamAllowNull(Object param, String paramName)
    * @param paramName Name of the param.
    * @throws Exception the exception
    */
-  public static void validateParam(Object param, String paramName)
-      throws Exception {
-    boolean isValid = false;
+  public static void validateParam(Object param, String paramName) throws Exception {
+    boolean isValid;
 
-    if (param != null && param instanceof String) {
+    if (param instanceof String) {
```

Comment:
I think we will have a problem here if the param is null which seems to be valid when looking at the compete block
```java
 if (param instanceof String) {
      String strParam = (String) param;
      isValid = !strParam.isEmpty();
    } else {
->    isValid = param != null;
    }
```